server/athenian/api/internal/features/entries.py

```diff
@@ -1858,6 +1877,7 @@ class TeamSpecificFilters:
     team_id: int
     repositories: Sequence[str]
     participants: PRParticipants | ReleaseParticipants | JIRAParticipants
+    jira_filter: JIRAFilter = JIRAFilter.empty()
```

Comment:
Well, the `repositories` doesn't have a default value. Add an empty tuple there? Or remove this default.